src/linter/ui5Types/fix/EventHandlersFix.ts

```diff
@@ -134,20 +122,20 @@ export default class EventHandlersFix extends XmlEnabledFix {
 	}
 
 	getAffectedSourceCodeRange() {
-		if (this.startPos === undefined || this.endPos === undefined) {
-			throw new Error("Start and end position are not defined");
+		if (this.startPos === undefined) {
+			throw new Error("Start position is not defined");
 		}
 		return {
 			start: this.startPos,
-			end: this.endPos,
+			end: 0,
```

Comment:
`0` doesn't seem right to me. Shouldn't this be `this.startPos + 1`, as we will insert one character?